.cloud_build/dart-services/dev.yaml

```diff
@@ -35,7 +35,6 @@ timeout: 1200s
 images:
   - '$_GCR_HOSTNAME/$PROJECT_ID/$REPO_NAME/$_SERVICE_NAME:$COMMIT_SHA'
 options:
-  machineType: N1_HIGHCPU_8
```

Comment:
It looks like the available machine types for cloud build are:

```
N1_HIGHCPU_8    Highcpu machine with 8 CPUs.
N1_HIGHCPU_32   Highcpu machine with 32 CPUs.
E2_HIGHCPU_8    Highcpu e2 machine with 8 CPUs.
E2_HIGHCPU_32   Highcpu e2 machine with 32 CPUs.
```

and:

`The default machine type is 1 CPU. Requesting a high-CPU virtual machine may increase the startup time of your build.`

Reply:
Its probably not critical, it would just be nice to use the same machine type for all the builds. If the builds do take that long perhaps a slightly faster machine (like `E2_HIGHCPU_8`) would improve things.